ssv/spectest/generate/state_comparison/*tests.MsgProcessingSpecTest/propose blinded decide regular.json

```diff
@@ -79,12 +79,12 @@
 										},
 										"ID": "AAADAY6ABmVRqBsxglhwntr33R9jzWhqDk24spu7es/mVghnevWlJ9lEjuR4NUheArULwAIAAAA=",
 										"Round": 1,
-										"Height": 0,
+										"Height": 5193728,
```

Comment:
why such an extreme height?

Reply:
When we did the Capella support we only changed sanity tests due to time constraints.
So I took the oppertunity to when I went over tests to change Bellatrix to Capella.
This use's Prater's capella slot.

However I am thinking on reverting those changes now to use Bellatrix. This is because:
1. It is not supposed to be in the scope of the PR.
2. Deneb is around the corner and I have a little plan. It maybe actually easier to upgrade to Deneb if we don't change to Capella now (but this is out of scope of the PR)

Reply:
Capella changes reverted